.github/workflows/release.yml

```diff
@@ -227,26 +212,34 @@ jobs:
           registry-url: 'https://registry.npmjs.org'
           node-version: ${{ env.node_version }}
 
-      - name: Download darwin node files
+      - name: Download mac node files
         uses: actions/download-artifact@v2.1.0
         with:
-          name: omega_edit_darwin.node
-          path: module/omega_edit/omega_edit_darwin.node
+          name: omega_edit_mac.node
+          path: module/omega_edit/omega_edit_mac.node
 
       - name: Download linux node files
         uses: actions/download-artifact@v2.1.0
         with:
           name: omega_edit_linux.node
           path: module/omega_edit/omega_edit_linux.node
 
+      - name: Download windows node files
+        uses: actions/download-artifact@v2.1.0
+        with:
+          name: omega_edit_win.node
+          path: module/omega_edit/omega_edit_win.node
+
       - name: Move out node files
         run: |
           cd module/omega_edit/
-          mv omega_edit_linux.node omega_edit_linux.node_dir
-          mv omega_edit_darwin.node omega_edit_darwin.node_dir
-          mv omega_edit_linux.node_dir/omega_edit_linux.node .
-          mv omega_edit_darwin.node_dir/omega_edit_darwin.node .
-          rm -rf omega_edit_linux.node_dir omega_edit_darwin.node_dir
+
+          for file in omega_edit_linux.node omega_edit_mac.node omega_edit_win.node;
+          do
+            mv $file "${file}_dir"
+            mv "${file}_dir/$file" .
+            rm -rf "${file}_dir"
+          done
```

Comment:
Why is this necessary?

Reply:
This is necessary because when downloading the artifacts they download into a folder so the folder structure would be `omega_edit_linux.node/omega_edit_linux.node`. So to be able to move out the file you need to rename the directory before you can move out the file. If you just do `mv omega_edit_linux.node/omega_edit_linux.node omega_edit_linux.node` you will get an error of location is directory not file.